### ISSUE TITLE
exception in RPC call

### ISSUE THREAD

**Issue by jungle-boogie** [borgbackup/borg#653]
Hello,

What in the world is causing this:
https://gist.github.com/jungle-boogie/fc2d6f4539b6038cea4f

The script isn't that special:
https://gist.github.com/jungle-boogie/031d4d0a0bfc2d7273cc

**Comment by jungle-boogie** [comment]
I believe this **may** be because I had one of the backups mounted with `mount`.

However, dismounting and running again, I still see this:
https://gist.github.com/jungle-boogie/13c8a79862a132be39bd

**Comment by ThomasWaldmann** [comment]
The last one looks like a corrupted repo. Try `borg check --repair`.

**Comment by jungle-boogie** [comment]
Hi @ThomasWaldmann,

I believe you're right on both--can't be mounted and corrupted.

As for the corruption, I don't what would have caused it but this is what I did.

Just before the errors, I was attempting to add more `--exclude`` statements before I knew I could pass a file (as seen in the gist above). I had a few (about 3) that were pointing to specific directories:

```
	--exclude /usr/local/etc/ssh/'ssh_host_*_key'            \
	--exclude '/usr/local/etc/ssh/ssh_host_key'               \
	--exclude /usr/local/etc/sasldb2.db
```

I then changed this: `/home/sean/RCS` to just ` /home/sean/` and `--exclude /home/sean/fossil-repos/`

That's when I noticed the errors. I believe the backup that morning ran fine as I got the cron email report from it.

To correct this, I just deleted (`rm -rf`) the directory on the machine where its backed up. I could not `borg delete` the entire repo with all of its backups. Since I'm not too worried about the history and now I'm backing up more data, the new backup will be better anyway,
